pynest/examples/CampbellSiegert.py

```diff
@@ -34,6 +34,11 @@
 Sven Schrader, Nov 2008, Siegert implementation by Tom Tetzlaff
 '''
 
+'''
+First, we import all necessary modules for simulation and analysis. Scipy
```

Comment:
On my computer, putting the two scipy imports after nest does not lead to any kind of trouble, can you post what the error message you get is?
(python 3.6.4, scipy 0.19)

Reply:
The original issue we experienced was https://github.com/numpy/numpy/issues/2521, apparently fixed in https://github.com/numpy/numpy/commit/a226daa75816577ac9c734133e600e5eebd9aaa9.

As the suggested import order does not matter in most cases and prevents a crash in the best case, I like the proposed addition although most people will not be hit by the long-fixed issue anymore.